Menu/menu_select_item.asm

```diff
@@ -33,7 +33,7 @@ Menu_AddressIndex:
   db $7EF34C ; shovel 7EF34F
   db $7EF34E ; Book 
   db $7EF350 ; Cane of Somaria
-  db $7EF351 ; Cane of Byrna
+  db $7EF351 ; Cane of Byrna / Fishing rod

```

Comment:
Let's correct this to reflect the Somaria and Byrna sharing a slot

Reply:
Done